### PR TITLE
Form (see commit comments)

### PR DESCRIPTION
**Pull request by CharleneDR**
Previous PR closed, comments taken into account, and other modifications added

### CODE REVIEW

File: src/View/layout.html.twig [hunk 1]
```diff
@@ -18,7 +18,7 @@
 		<body>
 			<nav class="navbar navbar-expand-lg">
 				<div class="container-fluid">
-					<a class="navbar-brand" href="#">
+					<a class="navbar-brand" href="/../home">
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:21]: The double dot are not needed, it should be /home

File: src/View/layout.html.twig [hunk 2]
```diff
@@ -46,7 +46,7 @@
 								<a class="nav-link link-dark active" aria-current="page" href="/../games?page=1">Les jeux</a>
 							</li>
 							<li class="mb-1">
-								<a class="nav-link link-dark active" aria-current="page" href="#">Mon compte</a>
+								<a class="nav-link link-dark active" aria-current="page" href="/../myaccount?view=0">Mon compte</a>
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:49]: same here

File: src/View/layout.html.twig [hunk 3]
```diff
@@ -61,22 +61,24 @@
 				{% block javascript %}{% endblock %}
 
 				<div class="container-fluid text-center">
-					<footer class="footer row d-flex align-items-center" style="background-color: #8CBDB9">
+					<footer class="footer row d-flex align-items-center">
 
 						<p class="col-md-4 mt-4 mb-2">
-							<img src="/assets/images/logo.png" class="img-fluid logo" alt="logo site">
+							<a href="/../home">
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:67]: Yep here too

File: src/View/layout.html.twig [hunk 4]
```diff
@@ -61,22 +61,24 @@
...
+								<img src="/assets/images/logo.png" class="img-fluid logo" alt="logo site">
+							</a>
 						</p>
 
 						<div class="col-md-4 mt-4 mb-2">
 							<ul class="list-unstyled mb-0 text-center">
 								<li class="mb-1">
-									<a href="#" class="nav-link px-2 link-dark">Accueil</a>
+									<a href="/../home" class="nav-link px-2 link-dark">Accueil</a>
```
Comment (aldaitzwild) [inline src/View/layout.html.twig:75]: Ok you get it

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/layout.html.twig (hunk 4 above)
Comment (CharleneDR) [reply src/View/layout.html.twig]: I got it :)

File: src/View/layout.html.twig (hunk 1 above)
Comment (CharleneDR) [reply src/View/layout.html.twig:21]: Done !

File: src/View/layout.html.twig (hunk 2 above)
Comment (CharleneDR) [reply src/View/layout.html.twig]: Done !

File: src/View/layout.html.twig (hunk 1 above)
Comment (aldaitzwild) [reply src/View/layout.html.twig:21]: Nope

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: src/View/layout.html.twig (hunk 1 above)
Comment (CharleneDR) [reply src/View/layout.html.twig]: :( Now it's ok !

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**